cumulus/parachains/integration-tests/emulated/common/src/xcm_helpers.rs

```diff
@@ -13,7 +13,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Cumulus
 use parachains_common::AccountId;
+
+// Polkadot
```

Comment:
very very nit, I would just remove docs for import, why is that important? 
I guess `rustfmt` could reorder anyway. And if we move xcm out of `polkadot-sdk` I am almost sure nobody will update this.

Reply:
What do you mean? the `// Cumulus`, `// Polkadot` and `// Substrate` comments?
I do not think they are making any bad, it helps to know what is actually imported and from where. It is helpful when comparing to the `Cargo.toml` (where the imports are split in the same way), where is pretty common to forget to delete unused crates.